### ISSUE TITLE
fix get_gg and get_silva makefile directives

### ISSUE THREAD

**Issue by thermokarst** [caporaso-lab/pretrained-feature-classifiers#12]
These are leftover from the pre-rescript days, and aren't being used anymore in this project, to the best of my knowledge.

Edit, I forgot that #10 was never completed - that's what I had in mind when I opened this issue, the original intent of that PR was to clean up the data fetching steps, among other things but I see now that a) that work was never finished and b) the work that _was_ completed was never merged.

cc @colinbrislawn

**Comment by thermokarst** [comment]
xref https://github.com/bokulich-lab/RESCRIPt/issues/124

**Comment by colinbrislawn** [comment]
As far as I can tell these are still needed to populate the ./input folder, as train.sh only takes .qza files as input.

get_silva.sh uses `qiime rescript get-silva-data` but get_gg.sh uses wget for everything, thus the cross ref.

Am I missing something? Is there a newer version in production?

**Comment by nbokulich** [comment]
@colinbrislawn is correct, these are still relevant to this project, `get_silva.sh` just uses `rescript` now.

Even with RESCRIPt, the data download might not be necessary with each QIIME 2 release (as it only needs to be run each time a new database version is released). Hence keeping it separate.

**Comment by thermokarst** [comment]
@colinbrislawn, yes, there is a newer version in production, looks like the dangling draft PR is still outstanding, here:

https://github.com/caporaso-lab/pretrained-feature-classifiers/pull/10
One of the pieces that I started working on while setting that PR up was either inlining the get_* steps, or, fixing up the makefile to properly declare the "order of operations." I have no real preference, I suppose, but as it stands I think we need to do _something_ different (right now that PR just settles on documenting the steps more clearly).

I edited my original issue above to clarify where my mind was at when I filed this issue, let me know if that makes more sense.

**Comment by thermokarst** [comment]
(fixing the makefile dependency steps is probably the easiest option here - ill update this issue title to make that more clear)
